internal/transform/liquidity_pool.go

```diff
@@ -36,15 +36,15 @@ func TransformPool(ledgerChange ingest.Change) (PoolOutput, error) {
 	}
 
 	var assetAType, assetACode, assetAIssuer string
-	err = cp.Params.AssetA.Extract(&assetAType, &assetACode, &assetAIssuer)
+	err = cp.Params.AssetA.Extract(&assetACode, &assetAIssuer, &assetAType)
```

Comment:
`Extract` actually takes this form: 
```
 Extract(typ interface{}, code interface{}, issuer interface{})
```

See [source code](https://pkg.go.dev/github.com/stellar/go/xdr#Asset.Extract)